modules/dxp/apps/search-experiences/search-experiences-service/src/main/resources/com/liferay/search/experiences/internal/model/listener/dependencies/limit_search_to_these_sites.json

```diff
@@ -29,9 +29,9 @@
 				{
 					"fields": [
 						{
-							"helpText": "group-ids-help",
-							"label": "group-ids",
-							"name": "scope_group_ids",
+							"helpText": "sites-external-reference-codes-help",
+							"label": "sites-external-reference-codes",
+							"name": "scope_group_external_reference_codes",
```

Comment:
Now we are changing these keys, would it make sense to make them consistent? Labels are `sites-xyz` and name is `scope_group_`external_reference_codes?